internal/controllers/ingress_controller.go

```
@@ -26,11 +26,12 @@ import (
 	"fmt"
 
 	"github.com/go-logr/logr"
+	corev1 "k8s.io/api/core/v1"
```

golangci-lint (2.12.2)

[error] ST1019: package "k8s.io/api/core/v1" is being imported more than once

(staticcheck)